### ISSUE TITLE
Constraint violation in 598. Range Addition II

### ISSUE THREAD

**Issue by tharun49** [LeetCode-Feedback/LeetCode-Feedback#4641]
#### Your LeetCode username

tharun49

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [x] Missing Test Cases 

#### Description of the bug

In the constraint it is mentioned that ops length varies from 1 to 10^4.
But when I tested with ops length as empty. System accepted and gave the output instead of erroring out

#### Code you used for Submit/Run operation

```
class Solution(object):
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        mat = [[0 for y in range(n)] for x in range(m)]
        for x in ops:
            for i in range(0,x[0]):
                for j in range(0, x[1]):
                    mat[i][j] += 1
        mn = 0
        for x in mat:
            if max(x) > mn:
                mn= max(x)
        res= 0
        for x in mat:
            res += x.count(mn)
            
        return res
```

#### Language used for code

Python

#### Expected behavior

Either the constraint should be changed or appropriate erroring out need to be done!

#### Screenshots

![LI](https://user-images.githubusercontent.com/88381274/131308105-d3b3670d-52c3-4af1-980f-6ec96af3c3af.jpg)

#### Additional context

So please take appropriate action for this BUG. Thanks!!

**Comment by azhar1038** [comment]
Copy of #4640

**Comment by lc-kay** [comment]
@tharun49 Thank you for reporting this. This will be fixed with #4640. Have a great day!
